test/correctness/simd_op_check.cpp

```diff
@@ -1455,23 +1460,23 @@ int main(int argc, char **argv) {
         in_i64 = ImageParam(Int(64), 1, "in_i64"),
         in_u64 = ImageParam(UInt(64), 1, "in_u64")
     };
-
-    for (ImageParam p : image_params) {
+    if (can_run_code()) {
```

Comment:
It took me a while to figure out why this matters. The problem you are fixing is different, which is that this test attempts to call realize, which will fail if you are on a platform that doesn't support JITing at all. can_run_code doesn't change this specifically. Instead, it checks that the host target matches the target being checked, which just happens to do what you want in our case because the host is x86 and the target is Hexagon.

I think maybe the better fix is to just not use realize to generate the random inputs.